roles/elastic-stack/ansible-kibana/tasks/main.yml

```diff
@@ -119,9 +131,9 @@
 
 
 - name: Install Wazuh Plugin (can take a while)
-  shell: "/usr/share/kibana/bin/kibana-plugin install {{ wazuh_app_url }}-{{ wazuh_version }}_{{ elastic_stack_version }}.zip"
-  environment:
-    NODE_OPTIONS: "--max-old-space-size=3072"
+  shell: >-
+    NODE_OPTIONS=" {{ node_options }} " /usr/share/kibana/bin/kibana-plugin install
```

Comment:
Same here

Reply:
The extra spaces have been removed on https://github.com/wazuh/wazuh-ansible/pull/385/commits/84b5510e3010f88da3863a53d416c789786fbded

Reply:
The extra spaces have been removed on https://github.com/wazuh/wazuh-ansible/pull/385/commits/84b5510e3010f88da3863a53d416c789786fbded